Libraries/Utilities/codegenNativeComponent.js

```diff
@@ -34,15 +34,14 @@ function codegenNativeComponent<Props>(
   componentName: string,
   options?: Options,
 ): NativeComponentType<Props> {
-  const errorMessage =
-    "Native Component '" +
-    componentName +
-    "' that calls codegenNativeComponent was not code generated at build time. Please check its definition.";
   if (global.RN$Bridgeless === true) {
+    const errorMessage =
+      "Native Component '" +
+      componentName +
+      "' that calls codegenNativeComponent was not code generated at build time. Please check its definition.";
```

Comment:
Thanks for sending this over @tido64 
I'm unsure this is the correct fix. Let me loop this internally.

Reply:
This looks like safe to land. @fortmarek we should cherry-pick this for the next RC 👍